examples/browser/react-app/src/index.tsx

```diff
@@ -3,26 +3,7 @@ import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
-import * as amplitude from '@amplitude/analytics-browser';
 
-/**
```

Comment:
Rather than changing this example to use the local snippet I think it might be better to create a new example e.g. `browser/react-app-snippet-spa` so we have examples with both usages. wdyt?

Reply:
yes, planning to add a new example in the playroom.